tests/cpp/test_fusion_profiler.cpp

```diff
@@ -25,6 +26,11 @@ class FusionProfilerTest : public NVFuserTest {
  protected:
   void SetUp() override {
     NVFuserTest::SetUp();
+
+    if (detectComputeSanitizer()) {
```

Comment:
Completely disables the FusionProfiler tests when compute-sanitizer is used. This should be fine as we should also run the tests without compute-sanitizer.